### DISCUSSION TITLE
Alternative to dropdown resource selection in Dashboard

### DISCUSSION THREAD

**Discussion by espenrl** [microsoft/aspire#3834] [Ideas]
Hi, would it be an idea to create buttons for each resource (possibly with unique colors) to better separate the resources and be more friendly for phone usage. Dropdowns are usable, but requires extra clicks when used (one to open dropdown, one to select resource).

In the image the squares are supposed to be a button for each resource.
 
<img width="765" alt="image" src="https://github.com/dotnet/aspire/assets/4621581/914def97-a8d9-4293-8319-2db086191444">

I guess this solution can lose its value if the list of resources is huge. Perhaps a cut of between a small and a large number of resources can be used. For small lists use buttons, and for large lists use a dropdown.

**Comment by davidfowl** [comment]
> I guess this solution can lose its value if the list of resources is huge.

Right, this doesn't seem like a solution that will scale.
